Makefile

```diff
@@ -1,32 +1,38 @@
-GIT_HASH := $(shell git log --format='%h' -n1)
 TRAVIS_BRANCH ?= $(shell git branch| grep \* | cut -d' ' -f2)
 BRANCH=$(TRAVIS_BRANCH)
 
+ifdef $(TRAVIS_PULL_REQUEST_BRANCH)
+ifneq ($(TRAVIS_PULL_REQUEST_BRANCH), "")
+	BRANCH = $(TRAVIS_PULL_REQUEST_BRANCH)
+endif
+endif
+
 flags=.makeFlags
 VPATH=$(flags)
 $(shell mkdir -p $(flags))
-dockerRepo=hashcloak
 
+dockerRepo=hashcloak
 katzenServerRepo=https://github.com/katzenpost/server
 katzenServerTag=$(shell git ls-remote --heads $(katzenServerRepo) | grep master | cut -c1-7)
 katzenServer=$(dockerRepo)/katzenpost-server:$(katzenServerTag)
 
 katzenAuthRepo=https://github.com/katzenpost/authority
-katzenAuthTag=$(shell git ls-remote --heads https://github.com/katzenpost/authority  | grep master | cut -c1-7)
+katzenAuthTag=$(shell git ls-remote --heads $(katzenAuthRepo) | grep master | cut -c1-7)
 katzenAuth=$(dockerRepo)/katzenpost-auth:$(katzenAuthTag)
 
 gethVersion=v1.9.9
 gethImage=$(dockerRepo)/client-go:$(gethVersion)
 mesonServer=$(dockerRepo)/meson
-mesonClient=$(dockerRepo)/meson-client
+hashcloakAuth=$(dockerRepo)/authority
 
-messagePush="LOG: Image already exists in docker.io/$(repo). Not pushing: "
+messagePush="LOG: Image already exists in docker.io/$(repoRepo). Not pushing: "
```

Comment:
`repoRepo` isn't defined anywhere else in this makefile. What is it suppose to point to? Also, why name it `repoRepo`? Did the previous name `repo` conflict with a previous make variable?

Reply:
I used the wrong variable. The correct one is `dockerRepo`